mdx/api/index.mdx

```diff
@@ -0,0 +1,776 @@
+  "value": "1200000",
+  "gasPrice": "8",
+  "protocolFee": "1200000",
+  "gas": "0",
+  "makerAssetAmount": "10000000",
+  "takerAssetAmount": "69348",
+  "orders": [
+    {
+      "chainId": 1,
+      "exchangeAddress": "0x61935cbdd02287b511119ddb11aeb42f1593b7ef",
+      "makerAddress": "0xb0dd0dcada73b715e3b5567ab17bd7f37216a637",
+      "makerAssetData": "0xf47261b00000000000000000000000006b175474e89094c44da98b954eedeac495271d0f",
+      "makerFeeAssetData": "0x",
+      "makerAssetAmount": "1199999999999999999941",
+      "makerFee": "0",
+      "takerAddress": "0x0000000000000000000000000000000000000000",
+      "takerAssetData": "0xf47261b0000000000000000000000000c02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
+      "takerFeeAssetData": "0x",
+      "takerAssetAmount": "8321716448330254529",
+      "takerFee": "0",
+      "senderAddress": "0x0000000000000000000000000000000000000000",
+      "feeRecipientAddress": "0x0000000000000000000000000000000000000000",
+      "expirationTimeSeconds": "1576007903",
+      "salt": "1576007663002",
+      "signature": "0x1c3046cceaabdc8efe55f92bff5d96aacfa46c144bfc9e1c13beb5074651ef658661f5e9af72e3c708aba61c00f535f7ff0364d3fdf8985b14b367d1bc8b139b4c03",
+      "fillableTakerAssetAmount": "8321716448330254529",
+      "fillableMakerAssetAmount": "1199999999999999999941",
+      "fillableTakerFeeAmount": "0"
+    }
+  ]
+}
+```
+</CodeTabs>
+
+# SRA
+
+The documentation for the Standard Relayer API.
+
+## GET /sra/asset_pairs
+
+Retrieves a list of available asset pairs and the information required to trade them.
+
+<Notification type="alert">This endpoint is paginated.</Notification>
+
+### Request 
+
+If either `assetDataA` or `assetDataB` are specified, the response will only return pairs that include that asset.
+If both `assetDataA` and `assetDataB` are specified, the response will only return that pair.
+
+| Query Param  | Description                                                                                                                                                 |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| assetDataA   | (Optional) A filter on the type of asset your may want information about in the form of [asset data](/docs/guides/v3-specification#assetdata).              |
+| assetDataB   | (Optional) A filter on the type of asset your may want information about in the form of [asset data](/docs/guides/v3-specification#assetdata).              |
+
+
+### Response
```

Comment:
This was confusing to me -- it wasn't apparent enough that the response is enveloped in a pagination structure. I don't think the pagination banner is enough here.

Reply:
(Same with the request part, need a better way to show it supports pagination)

Reply:
@johnrjj how do you suggest we do that? Full pagination structure every time?

Reply:
no, i don't think so -- other docs show paginated records like this; i think it's just a usability thing and can be as simple as making Paginated bold or something, the user just needs a nudge when looking at the response (some responses return pagination others don't so it's easy to forget when new/unfamiliar).

Reply:
Hmmm ok. Will do some thinking.